src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.EcDsa.ImportExport.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Diagnostics;
+
+internal static partial class Interop
+{
+    internal static partial class Crypto
+    {
+        // todo: remove this
+        [DllImport(Libraries.CryptoNative, EntryPoint = "OutputDebugString")]
+        public static extern void OutputDebugString(string oidValue);
```

Comment:
FWIW, I often just do this when I need it with code like:
```C#
[DllImport("libc")]
private static extern int printf(string format, string);
...
printf("some %s\n", "text");
```
You can also use Debug.WriteLine, and if you sent the COMPlus_DebugWriteToStdErr environment variable to 1, WriteLine will write to stderr.